<launch>
  <node name="compensator" pkg="colab_reachy_calibrate" type="compensator.py" respawn="false" output="screen">
    <param name="right_error_map_file" value="$(find colab_reachy_calibrate)/config/ReachyRightArmErrorMap.json" />
    <param name="left_error_map_file" value="$(find colab_reachy_calibrate)/config/ReachyLeftArmErrorMap.json" />
  </node>
</launch>
